src/lib/prisma-session-store.spec.ts

```diff
@@ -6,6 +6,15 @@ import { createPrismaMock } from '../mocks';
 import { PrismaSessionStore } from './prisma-session-store';
 import { createExpiration, range, sleep } from './utils/testing';
 
+declare module 'express-session' {
+  // tslint:disable-next-line: naming-convention
+  interface SessionData {
+    sample?: boolean;
+    unrealizable?: string;
+    data?: string;
+  }
+}
+
```

Comment:
With the latest type definitions of `@types/express-session` this is how `SessionData` has it's types defined.
See: [@types/express-session/express-session-tests.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/da76ba1925ec673e67a26a3ddf5e43965d7ab185/types/express-session/express-session-tests.ts#L63-L68)